lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -73,39 +73,150 @@
-            }
-        ],
-        "image": {
-            "offer": "CoreOS",
-            "publisher": "CoreOS",
-            "sku": "Stable",
-            "version": "899.17.0"
-        },
-        "load_balancer": null,
-        "location": "eastus",
-        "managed_disk_type": "Standard_LRS",
-        "name": "testVMSSeb4fd3c704",
-        "os_disk_caching": "ReadWrite",
-        "os_type": "Linux",
-        "resource_group": "myresourcegroup",
-        "ssh_password_enabled": false,
-        "state": "present",
-        "subnet_name": null,
-        "tier": "Standard",
-        "upgrade_policy": "Manual",
-        "virtual_network_name": null,
-        "vm_size": "Standard_DS1_v2"
-    }]
+    type: complex
+    contains:
+        admin_username:
+            description:
+                - Admin username used to access the host after it is created.
+            returned: always
+            type: str
+            sample: adminuser
+        capacity:
+            description:
+                - Capacity of VMSS.
+            returned: always
+            type: int
+            sample: 2
+        data_disks:
+            description:
+                - List of attached data disks.
+            returned: always
+            type: complex
+            contains:
+                caching:
+                    description:
+                        - Type of data disk caching.
+                    type: str
+                    sample: ReadOnly
+                disk_size_gb:
+                    description:
+                        - The initial disk size in GB for blank data disks
+                    type: int
+                    sample: 64
+                lun:
+                    description:
+                        - The logical unit number for data disk
+                    type: int
+                    sample: 0
+                managed_disk_type:
+                    description:
+                        - Managed data disk type
+                    type: str
+                    sample: Standard_LRS
+        image:
+            description:
+                - Image specification
+            returned: always
+            type: complex
+            contains:
+                offer:
+                    description:
+                        - Offer.
+                    type: str
+                    sample: RHEL
+                publisher:
+                    description:
+                        - Publisher name.
+                    type: str
+                    sample: RedHat
+                sku:
+                    description:
+                        - SKU name.
+                    type: str
+                    sample: 7-RAW
+                version:
+                    description:
+                        - Image version.
+                    type: str
+                    sample: 7.5.2018050901
+        load_balancer:
+            description:
+                - Load balancer name.
+            returned: always
```

Comment:
there's vmss without loadbalancer. not always return

Reply:
it's returned, just null